src/components/MemoryViewer/MemoryViewer.scss

```diff
@@ -0,0 +1,113 @@
+@import '../../styles/mixins.scss';
+
+$memory-type-colors: (
+    'AllocatorCachesMemory': var(--g-color-base-utility-medium-hover),
+    'SharedCacheConsumption': var(--g-color-base-info-medium-hover),
+    'MemTableConsumption': var(--g-color-base-warning-medium-hover),
+    'QueryExecutionConsumption': var(--g-color-base-positive-medium-hover),
+    'Other': var(--g-color-base-neutral-light-hover),
+);
+
+@mixin memory-type-color($type) {
+    background-color: map-get($memory-type-colors, $type);
+}
+
+.memory-viewer {
```

Comment:
Progress bar is bigger, than other progress-bars, text isn't aligned in the middle vertically

![Screenshot 2024-11-12 at 16 21 08](https://github.com/user-attachments/assets/970a8d45-62ab-4651-b76d-4300ac66e19e)

Reply:
@kunga requested it to be as big as possible

Reply:
Ah, okey)